File: filcnaplo_premium/android/widget_timetable/WidgetTimetableService.java
package hu.filc.naplo.widget_timetable;

import android.content.Intent;
import android.os.Build;
import android.widget.RemoteViewsService;

public class WidgetTimetableService extends RemoteViewsService {
    @Override
    public RemoteViewsFactory onGetViewFactory(Intent intent) {
        return new WidgetTimetableDataProvider(getApplicationContext(), intent);
    }
}

File: refilc/android/app/src/main/java/hu/refilc/naplo/widget_timetable/WidgetTimetableService.kt
package hu.refilc.naplo.widget_timetable

import android.content.Context
import android.content.Intent
import android.os.Build
import android.widget.RemoteViewsService

class WidgetTimetableService: RemoteViewsService() {
  override fun onGetViewFactory(intent: Intent): RemoteViewsFactory {
    return WidgetTimetableDataProvider(getApplicationContext(), intent)
  }
}
